tests/test_p2p_session.rs

```diff
@@ -115,7 +115,7 @@ fn test_advance_frame_p2p_sessions() -> Result<(), GGRSError> {
     assert!(sess1.current_state() == SessionState::Synchronizing);
     assert!(sess2.current_state() == SessionState::Synchronizing);
 
-    for _ in 0..10 {
+    for _ in 0..50 {
```

Comment:
@gschup I would be happy to revert that, but on a 2015 macos, even `0..30` lead to sometimes failures. I figured it's a small price to pay to make the tests a little bit more stable. Eventually we would probably want a timeout, but it's too far from this Pull Request scope.

Reply:
I find this to be very odd. This never was an issue for me, but network packages are mysterious :D